### ISSUE TITLE
Sending A Sticker

### ISSUE THREAD

**Issue by insideheartz** [pedroslopez/whatsapp-web.js#358]
probably this is duplicate , but i tried to send a sticker in this library 
but i get some problem to implement this code for injected.js
```javascript
window.WWebJS.sendSticker = async (sticker, chatId, metadata, type) =>{
    var chat = await window.Store.Chat.get(chatId);

    if(chat.erro === false || chat.__x_id){
    var ListChat = await Store.Chat.get(chatId),
           stick = new window.Store.Sticker.modelClass();

    stick.__x_clientUrl = sticker.clientUrl;
    stick.__x_filehash = sticker.filehash;
    stick.__x_id = sticker.filehash;
    stick.__x_uploadhash = sticker.uploadhash;
    stick.__x_mediaKey = sticker.mediaKey;
    stick.__x_initialized = false;
    stick.__x_mediaData.mediaStage = 'INIT';
    stick.mimetype = 'image/webp';
    stick.height = metadata && metadata.height ? metadata.height : 512;
    stick.width = metadata && metadata.width ? metadata.width : 512;
    
    await stick.initialize();
    
    var result = await (Promise.all((ListChat)? await stick.sendToChat(chat, {stickerIsFirstParty: false, stickerSendOrigin: 6}) : ""));
    result = result.join('');
    var m = {type: type},
    To = await WWebJS.getChat(chatId);
   
      }else{
          return chat;
      }
}	
```
that's show undefinied in console
or something wrong on that?

i mean like this
![jjj](https://user-images.githubusercontent.com/66521965/94431489-e1c49600-0184-11eb-9594-addf079f7c80.PNG)

hope this feature will available in this library

**Comment by mpirescarvalho** [comment]
Hey, check this [PR](https://github.com/pedroslopez/whatsapp-web.js/pull/479)

**Comment by pedroslopez** [comment]
Implemented in #479 (ff7586a164be82c68b7dcbb50bec67bcfe68ed4a) and will be released soon!
